packages/confect/src/server/ConfectQueryInitializer.ts

```diff
@@ -45,7 +53,7 @@ type ConfectQueryInitializer<
       ...indexFieldValues: IndexFieldTypesForEq<
         ConfectDataModel.ConfectDataModel.DataModel<ConfectDataModel_>,
         TableName,
-        Indexes<_TableInfo>[IndexName] & GenericIndexFields
+        Indexes<_TableInfo>[IndexName]
```

Comment:
If this isn't necessary it's definitely better removed, but was it breaking anything?